app/javascript/utilities/i18n_support.js

```diff
@@ -0,0 +1,18 @@
+//Load the package
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+//Read the Yaml file
+const locale = './config/locales/en.yml';
+const data = fs.readFileSync(locale, 'utf8');
+const yamlData = yaml.load(data);
+
+document.body.innerHTML += `<div id="i18n-translations"></div>`;
+document.getElementById('i18n-translations').dataset.translations =
+  JSON.stringify(yamlData);
+
+export function i18nSupport() {
```

Comment:
Yep as part of the follow-up we could automatically load this as part of test setup